app/org/sagebionetworks/bridge/dynamodb/DynamoTaskDao.java

```diff
@@ -37,6 +39,11 @@ public void setTaskIndex(DynamoIndexHelper index) {
         this.index = index;
     }
     
+    @Resource(name = "taskEventService")
+    public void setTaskEventService(TaskEventService taskEventService) {
```

Comment:
Minor: Final, in the unlikely event we subclass DynamoTaskDao and override the Spring setter.